time stamp
float64 dt
float64 d_dist
float64 d_th
float64 v
float64 w
